libs/blocks/graybox-promote/components/graybox-promote-lit.js

```diff
@@ -1,7 +1,7 @@
 import { getSheet } from '../../../../tools/utils/utils.js';
 import { LitElement, html } from '../../../deps/lit-all.min.js';
 import { Task } from '../../../deps/lit-task.min.js';
-import login from '../../../tools/sharepoint/login.js';
+// import login from '../../../tools/sharepoint/login.js';
```

Comment:
Is this still needed here?

Reply:
Yes, I'll need it for my next ticket. I commented it out to avoid the eslint check failing.